device/celestica/x86_64-cel_belgite-r0/pmon_daemon_control.json

```diff
@@ -0,0 +1,4 @@
+{
+    "skip_ledd": true,
+    "skip_pcied": true
```

Comment:
is there any specific reason that you need to skip the pcied for this platform?

Reply:
Hi Jinkang,   Please ignore this first,  I am doing the verification in my local.  also I will update it later. 
By the way, seems I have encountered some issue in the validation. 
 seems the latest code in master branch using the python 3.9 as the following shows:
root@sonic:/# find -name psu.py
./host/image-Add_Belgite_support.0-dirty-20220209.082924/rw/usr/local/lib/python                                                                             3.9/dist-packages/sonic_platform/psu.py
./host/image-Add_Belgite_support.0-dirty-20220209.082924/docker/overlay2/ff5ee5f                                                                             19a80d36a902574999c1324886c537568bd9e079402545b39a5af5e33/diff/usr/local/lib/pyt                                                                             hon3.7/dist-packages/sonic_platform/psu.py
find: ‘./proc/18961/task/18961/net’: Invalid argument
find: ‘./proc/18961/net’: Invalid argument
find: ‘./proc/18962/task/18962/net’: Invalid argument
find: ‘./proc/18962/net’: Invalid argument
./usr/local/lib/python3.9/dist-packages/sonic_platform/psu.py


but our thermalctld still import the python3.7 's module, which as the syslog shows, do we update the python version 
in SONiC  codebase?  seems the platform API verification is ok in branch 202106. 

Jul 25 03:16:30.677708 sonic INFO systemd[1]: Condition check resulted in gbsyncd service being skipped.
Jul 25 03:16:30.708447 sonic INFO teamd.sh[6747]: Creating new teamd container with HWSKU CELESTICA-BELGITE
Jul 25 03:16:30.732821 sonic INFO pmon#/supervisord: psud Traceback (most recent call last):
Jul 25 03:16:30.736414 sonic INFO pmon#/supervisord: psud   File "/usr/local/lib/python3.7/dist-packages/sonic_platform/platform.py", line 10, in <module>
Jul 25 03:16:30.736414 sonic INFO pmon#/supervisord: psud     from sonic_platform_pddf_base.pddfparse import PddfParse 
Jul 25 03:16:30.736414 sonic INFO pmon#/supervisord: psud ModuleNotFoundError: No module named 'sonic_platform_pddf_base.pddfparse'
Jul 25 03:16:30.736414 sonic INFO pmon#/supervisord: psud 
Jul 25 03:16:30.736414 sonic INFO pmon#/supervisord: psud During handling of the above exception, another exception occurred:
Jul 25 03:16:30.736414 sonic INFO pmon#/supervisord: psud 
Jul 25 03:16:30.736414 sonic INFO pmon#/supervisord: psud Traceback (most recent call last):
Jul 25 03:16:30.736455 sonic INFO pmon#/supervisord: psud   File "/usr/local/bin/psud", line 17, in <module>
Jul 25 03:16:30.736455 sonic INFO pmon#/supervisord: psud     from sonic_platform.psu import Psu
Jul 25 03:16:30.736455 sonic INFO pmon#/supervisord: psud   File "/usr/local/lib/python3.7/dist-packages/sonic_platform/__init__.py", line 3, in <module>
Jul 25 03:16:30.736479 sonic INFO pmon#/supervisord: psud     from sonic_platform import *
Jul 25 03:16:30.736479 sonic INFO pmon#/supervisord: psud   File "/usr/local/lib/python3.7/dist-packages/sonic_platform/platform.py", line 14, in <module>
Jul 25 03:16:30.736505 sonic INFO pmon#/supervisord: psud     raise ImportError(str(e) + "- required module not found")
Jul 25 03:16:30.736505 sonic INFO pmon#/supervisord: psud ImportError: No module named 'sonic_platform_pddf_base.pddfparse'- required module not found
Jul 25 03:16:30.736505 sonic INFO pmon#supervisord 2021-07-25 03:16:30,731 INFO exited: psud (exit status 1; not expected)
Jul 25 03:16:30.770896 sonic DEBUG bgp#bgpcfgd: execute command '['vtysh', '-c', 'show bgp vrf default neighbors json']'.
Jul 25 03:16:30.788016 sonic INFO pmon#/supervisord: thermalctld Traceback (most recent call last):